package exception;

public class Example3 {

	public static void main(String[] args) {
		
		String s = "Bangalore";
		
		try
		{
			char c = s.charAt(20);
			System.out.println(c); // n
		}
		catch (RuntimeException e) {
			// TODO: handle exception
		}
		catch(StringIndexOutOfBoundsException e)
		{
			System.out.println("Exception1");
			e.printStackTrace();
		}
		catch(NullPointerException e)
		{
			System.out.println("Exception2");
			e.printStackTrace();
		}
		catch (ArithmeticException e) {
			System.out.println("Exception3");
			e.printStackTrace();
		}
		
		catch (Exception e) {
			// TODO: handle exception
		}
		catch (Throwable e) {
			// TODO: handle exception
		}
		
		try
		{
			int k = 10/0; // Arithmetic
			System.out.println(k);
		}
		catch (ArithmeticException e) {
			e.printStackTrace();
		}
		
		System.out.println("main method end...");
		
	}
}
